### PR TITLE
DB Migrate

### PR DESCRIPTION
**Pull request by maximstrukov**

### CODE REVIEW

File: bundle/Command/MigrateCommand.php [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+     * @var Repository;
+     */
+    private $ezRepository;
+
+    /**
+     * @var ConfigResolverInterface
+     */
+    private $configResolver;
+
+    public const CAMPAIGN_LIST_CONTENT_ID = 53;
```
Comment (Plopix) [inline bundle/Command/MigrateCommand.php]: this must me remove right? can we automatically fallback on content ID 1 if somehow the ID is not found?

File: bundle/Command/MigrateCommand.php [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+        $this->entityManager->flush();
+
+        $this->io->section(
+            'Total: '.$listCounter.' lists, '.$mailingCounter.' mailings, '.$userCounter.' users, '.
+            $subscriptionCounter.' registrations.'
+        );
+        $this->io->success('Import done.');
+    }
+
+    private function clean(): void
```
Comment (Plopix) [inline bundle/Command/MigrateCommand.php]: I think what you are doing `TRUNCATE` please check and change ;)

File: bundle/Command/MigrateCommand.php (hunk 1 above)
Comment (maximstrukov) [reply bundle/Command/MigrateCommand.php]: Changed to Content Id = 1

File: bundle/Command/MigrateCommand.php (hunk 2 above)
Comment (maximstrukov) [reply bundle/Command/MigrateCommand.php]: Don't use TRUNCATE because of FOREIGN KYES constraints. Added the comment about that

**Pull request merged**